packages/client-sdk-nodejs/test/integration/cache-client-close.test.ts

```diff
@@ -0,0 +1,23 @@
+import {expectWithMessage} from '@gomomento/common-integration-tests';
+
+import {GetBatch, ICacheClient} from '@gomomento/sdk-core';
+import {SetupIntegrationTestWithMiddleware} from './integration-setup';
+
+const {cacheClient, cacheName} = SetupIntegrationTestWithMiddleware();
+
+runTestWithClientCloseToMakeSureTestDoesntRunForever(cacheClient);
+export function runTestWithClientCloseToMakeSureTestDoesntRunForever(
```

Comment:
not necessarily a problem but I'm curious what the value is of having a function here rather than just having `describe` at the top level like it is for most tests?

Reply:
ah yeah i didn't put a lot of thought into that honestly will address